### DISCUSSION TITLE
Wizard One Column

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#2990] [Help]
hello guys, I don't know why this happens, when implementing the functionality of Tabs or Wizard within my Form, the view is reduced to a column, I attach an image as a reference. Any advice?

![imagen](https://user-images.githubusercontent.com/13526704/176698120-c2aec87b-13ce-4f7b-81cb-faa6cbfcbc01.png)

**Comment by invaders-xx** [comment]
Try to add your wizard in a Grid like : 
```php
Forms\Components\Grid::make(1)
                            ->schema([
                                Forms\Components\Wizard::make([])
                            ])->columns([
                                'sm' => 1,
                                'lg' => null,
                            ])
```

**Reply by jcgmalave** [reply]
Thanks a lot!!! IT WORKS!!!

**Comment by danharrin** [comment]
We have an extra docs for Create pages... doing it properly will set the correct width & move the submit button to the end of the wizard

https://filamentphp.com/docs/2.x/admin/resources/creating-records#wizards
